tests/integration/hub_usage/test_hub_usage.py

```diff
@@ -148,3 +151,13 @@ def test_hub_build_push_push_again(monkeypatch, mocker):
         # try and push same version again should fail with `--no-overwrite`
         args = set_hub_build_parser().parse_args([str(cur_dir) + '/hub-mwu', '--push', '--host-info', '--no-overwrite'])
         HubIO(args).build()
+
+
+@pytest.mark.timeout(360)
+@pytest.mark.parametrize('dockerfile_path',
+                         [os.path.join(cur_dir, 'hub-mwu-multistage'),
+                          os.path.relpath('hub-mwu-multistage', os.getcwd())])
```

Comment:
I think this line should be `os.path.relpath(os.path.join(cur_dir, 'hub-mwu-multistage'), os.getcwd())`, then it should work

Reply:
fixed